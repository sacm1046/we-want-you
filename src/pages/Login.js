import React from "react";
import Anchor from "../components/anchor/Anchor";
import Spotify from "../components/icons/Spotify";
import { anchorType } from "../utils/anchorTypes";
import { color } from "../utils/systemColors";

const { REACT_APP_SPOTIFY_CLIENT_ID } = process.env;
const URI = `https://accounts.spotify.com/authorize?client_id=${REACT_APP_SPOTIFY_CLIENT_ID}&response_type=code&redirect_uri=http://localhost:3000&%20user-read-email20%user-read-private20%user-library-read20%user-library-modify%20user-read-playback-state%20user-modify-playback-state`;

export default function Login() {
  return (
    <section className="section">
      <Spotify size={250} color={color.primary} />
      <Anchor href={URI} text="MUSIC NOW" className="mt-10" type={anchorType.default} />
    </section>
  );
}
