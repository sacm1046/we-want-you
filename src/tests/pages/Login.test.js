import React from "react";
import { shallow } from "enzyme";
import Login from "../../pages/Login";

describe("Test para pages/Login", () => {
  test("Se debe renderizar el componente Login", () => {
    const wrapper = shallow(<Login />);
    expect(wrapper).toMatchSnapshot();
  });
});
