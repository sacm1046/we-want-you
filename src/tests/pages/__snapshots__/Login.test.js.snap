// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test para pages/Login Se debe renderizar el componente Login 1`] = `
<section
  className="section"
>
  <Spotify
    color="#fff"
    size={250}
  />
  <Anchor
    className="mt-10"
    href="https://accounts.spotify.com/authorize?client_id=605c5be1972943f6b90dc2159807073e&redirect_uri=http://localhost:3000&scope=user-read-currently-playing%20user-read-recently-played%20user-read-playback-state%20user-top-read%20user-modify-playback-state%20streaming&response_type=token&show_dialog=true"
    text="MUSIC NOW"
    type="default"
  />
</section>
`;
